prac_05/word_occurences.py

```diff
@@ -8,3 +8,56 @@
 - Align the numbers so they are in one nice column (see prac for notes)
 """
 
+from operator import itemgetter
+
+
+def main():
+    # requests phrase from user
+    phrase = input("enter a phrase:")
+    new_dictionary = convert_to_dict(phrase)
+    longest_word = get_longest_word(new_dictionary)
+    sorting_list = initialize_sorting_list(new_dictionary)
+
+    # Sorting the list by key and then printing
+    sorting_list.sort()
+    print("Below is the dictionary sorted by key:")
+    for i in sorting_list:
```

Comment:
Why would `i` be a good name for a word? What would be a good name for a `word`?

Reply:
I am making the mistake of writing a place holder value while i write and then renaming to a more specific value later for the index. I think this is a good example for why i shouldnt do things this way. I'll fix this up!